src/components/__stories__/login_header.tsx

```diff
@@ -5,5 +5,5 @@ import LoginHeader from "../login_header"
 
 storiesOf("LoginHeader", LoginHeader)
   .add("Simple login header", () => (
-    <LoginHeader/>
+    <LoginHeader title={"Welcome back, please log in to your account"}/>
```

Comment:
@broskoski I think I halfway grokked that because I'm accessing the text for the Title component in the header from `this.props.children`. Thanks for clearing it up. Yuki's suggestions makes sense as far as how it all comes together.